doc/contributing/docs/style.rst

```diff
@@ -21,4 +21,19 @@ Wrong usage: "Replication allows multiple Tarantool *servers* to work on copies
 of the same databases."
 
 Correct usage: "Replication allows multiple Tarantool *instances* to work on
-copies of the same databases."
\ No newline at end of file
+copies of the same databases."
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Don't use i.e. and e.g.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Don't use the following contractions:
+
+*   `"i.e." <https://www.merriam-webster.com/dictionary/i.e.>`_
+    —from Latin "id est". Use "that is" or "which means" instead.
+*   `"e.g." <https://www.merriam-webster.com/dictionary/e.g.>`_
+    —from Latin "exempli gratia". Use "for example" instead.
+
+Many people, especially non-native English speakers,
+aren't familiar with `these contractions
+<https://www.merriam-webster.com/words-at-play/ie-vs-eg-abbreviation-meaning-usage-difference>`_.
```

Comment:
Correct usage should not be a problem for a reader I guess.

Reply:
People also misuse them, i.e. use "i.e." when they mean to say "for example". ;)

Reply:
That's true (for me as well before I read links you pointed here). However I'm about reading. Is that a problem?